lib/everypolitician/popolo/event.rb

```diff
@@ -3,6 +3,20 @@ module Popolo
     class Events < Collection; end
     class Event < Entity
       attr_accessor :start_date, :end_date
+
+      def people
+        raise NotImplementedError unless classification == 'legislative period'
+        ids = popolo.memberships.select { |m| m.legislative_period_id == id }
+                    .map(&:person_id)
```

Comment:
You probably want a `to_set` here, partially because people can have multiple Memberships in a Term, but also because calling `.include?` on it will be much faster than on an Array